### ISSUE TITLE
[Feature Request] Custom xdaythreshold for each coin? 

### ISSUE THREAD

**Issue by Winckelmann** [Mikadily/poloniexlendingbot#244]
The demand and supply for each market is different. I think it would be a good customisation, if it is possible.

For example, I would not mind lending ETH at 0.013% for 30days but I would prefer a higher  xdaysthreshold for BTC.

Edit: Custom xdays for each coin would be good too, for people who want to vary how long they would like to commit each coin at the higher rates.

**Comment by Evanito** [comment]
You want to use the coincfg: https://poloniexlendingbot.readthedocs.io/en/latest/configuration.html#config-per-coin

**Comment by laxdog** [comment]
Definitely possible and not too hard, but certainly non-trivial.

I'd be happy to look at it at some stage, but there's a lot of other stuff I'm planning on doing first.

It's as easy to do xdays as it is to do xdays and xdaysthreshold. So may as well do both.

I'd like to say though if this is done, it shouldn't be added to the coincfg dictionary, only the per coin config. We'd be breaking config backwards compatibility for something I don't think that many people would be using.

**Comment by rnevet** [comment]
I agree with laxdog, if we would do this I would like to have a more general solution for coin specific configuration overriding general config.   I would also remove the old coincfg config and only support one config method.
